README.md

```diff
@@ -6,32 +6,32 @@ This package allows one to use [conda](http://conda.pydata.org/) as a binary pro
 While other binary providers like [Hombrew.jl](https://github.com/JuliaLang/Homebrew.jl),
 [AptGet](https://en.wikipedia.org/wiki/Advanced_Packaging_Tool#apt-get) or
 [WinRPM.jl](https://github.com/JuliaLang/WinRPM.jl) are platform-specific,
-Conda.jl is a cross-platform alternative. It also also be used without administrator rights,
+Conda.jl is a cross-platform alternative. It can also be used without administrator rights,
 in contrast to the current Linux-based providers.
 
 `conda` is a package manager which started as the binary package manager
 for the Anaconda Python distribution, but it also provides arbitrary packages.
-Instead of the full Anaconda distribution, Conda.jl uses the miniconda minimalistic Python environment,
+Instead of the full Anaconda distribution, Conda.jl uses the miniconda Python environment,
 which only includes `conda` and its dependencies.
 
 You can install it by running `Pkg.add("Conda")` at the Julia prompt.
 
-## Basic functionalities
+## Basic functionality
```

Comment:
My english grammar might be faulty, but isn't the plural required here ? We are talking about the functionalities of the Conda module.